### PR TITLE
Smbclient utility module.

### PR DESCRIPTION
**Pull request by Jjk422**
Scenario file found at '/home/user/RubymineProjects/SecGen/scenarios/simple_examples/smbclient_utility.xml'

Installs smbclient for use with samba services.

### CODE REVIEW

File: scenarios/simple_examples/smbclient_utility.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<scenario xmlns="http://www.github/cliffe/SecGen/scenario"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.github/cliffe/SecGen/scenario">
+
+  <system>
+    <system_name>smbclinet_utility</system_name>
```
Comment (thomashaw) [inline scenarios/simple_examples/smbclient_utility.xml]: smbclinet_utility -> smbclient_utility

File: modules/utilities/unix/smb/smbclient/secgen_metadata.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<utility xmlns="http://www.github/cliffe/SecGen/utility"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.github/cliffe/SecGen/utility">
+  <name>Samba client</name>
+  <author>example42</author>
```
Comment (thomashaw) [inline modules/utilities/unix/smb/smbclient/secgen_metadata.xml]: is this an example42 module or does this need to go?

File: scenarios/simple_examples/smbclient_utility.xml (hunk 1 above)
Comment (Jjk422) [inline scenarios/simple_examples/smbclient_utility.xml]: This change seems fine.

File: modules/utilities/unix/smb/smbclient/secgen_metadata.xml (hunk 2 above)
Comment (Jjk422) [inline modules/utilities/unix/smb/smbclient/secgen_metadata.xml]: I must have copied the secgen_metadata.xml from another module and forgot to remove this author. Good spot.

**Comment by Jjk422** [comment]
All the changes should now have been made, hopefully this module can now be merged.

**Pull request merged**